janusgraph-core/pom.xml

```diff
@@ -37,6 +37,11 @@
             <artifactId>tinkergraph-gremlin</artifactId>
             <version>${tinkerpop.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tinkerpop</groupId>
+            <artifactId>gremlin-server</artifactId>
+            <version>${tinkerpop.version}</version>
```

Comment:
Make this dependency optional with `<optional>true</optional>`. Not all users of `janusgraph-core` use the `gremlin-server`, and I'm aware of one user that runs their own custom server. Setting this as optional will avoid introducing other downstream dependencies into core, like `netty-all`.

Note that you'll need to add a dependency into `janusgraph-test/pom.xml` for `gremlin-server` since it's required for the tests.